Gopkg.toml

```diff
@@ -44,7 +44,8 @@ required = [
 
 [[constraint]]
   name = "github.com/portworx/sched-ops"
-  branch = "operator"
+  source = "github.com/piyush-nimbalkar/sched-ops"
```

Comment:
Was this intentional ?

Reply:
It was intention, to get the PR started as the sched ops PR was pending as well.
Updated now.